astropy/timeseries/periodograms/lombscargle_multiband/core.py

```diff
@@ -0,0 +1,709 @@
+"""Main Lomb-Scargle Multiband Implementation"""
+
+import numpy as np
+
+from astropy import units
+from astropy import units as u
+from astropy.time import Time, TimeDelta
+from astropy.timeseries.binned import BinnedTimeSeries
+from astropy.timeseries.periodograms.lombscargle import LombScargle
+from astropy.timeseries.sampled import TimeSeries
+
+from .implementations import available_methods, lombscarglemultiband
+from .implementations.mle import construct_regularization, design_matrix, periodic_fit
+
+
```

Comment:
This is to allow for it to just be `from .implementations.mle import *` , correct? The difficulty I see with that is that we get into trouble with Flake8, (https://www.flake8rules.com/rules/F405.html), where it the imports are undefined. If we don't care about violating Flake8 that's fine, but there does seem to be an argument to be made for keeping these imports explicit.

Reply:
Not necessarily to enable using ``*`` but more to also restrict what *can* be imported from modules. In general we define ``__all__`` in all files - this serves as a way to be explicit about what is 'public' API.

Reply:
This is to prevent users from importing say 'has_units' from this module, because they might do it and claim it was 'public API' because it worked and there were no underscores at the start of the name.

Reply:
I see, thanks for the clarification! Added an __all__ to the files that were missing it.